pinot-query-planner/src/main/java/org/apache/pinot/query/planner/logical/RexExpression.java

```diff
@@ -204,6 +204,7 @@ public FieldSpec.DataType getDataType() {
   class FunctionCall implements RexExpression {
     @ProtoProperties
     private SqlKind _sqlKind;
+    // TODO: Make sure _sqlKind matches _functionName.
```

Comment:
this is not necessarily needed. otherwise we won't need both in the RexExpression

Reply:
Can you clarify more on why it is not needed so that I can document it in the comment?

Reply:
e.g. remove this comment. this TODO is not needed. 
if you were to add comments to each individual member variables, one suggestion would be 
```
    // the underlying SQL operator kind of this function. It can be either a standard SQL operator or an extended function kind.
    // @see #SqlKind.FUNCTION, #SqlKind.OTHER, #SqlKind.OTHER_FUNCTION
    @ProtoProperties
    private SqlKind _sqlKind;

    // the return data type of the function.
    @ProtoProperties
    private FieldSpec.DataType _dataType;

    // the name of the SQL function. For standard SqlKind it should match the SqlKind ENUM name.
    @ProtoProperties
    private String _functionName;

    // the list of RexExpressions that represents the operands to the function.
    @ProtoProperties
    private List<RexExpression> _functionOperands;
```

Reply:
Added the comment.